docs/comparison.rst

```diff
@@ -0,0 +1,83 @@
+
+Comparison with other simulators
+================================
+
+pyuvsim is designed to calculate visibilities from point sources and a given primary beam model. It does this through direct calculation of the full-polarization Radio Interferometer Measurement Equation (RIME), fully parallelized across a set of MPI processes. Its development has progressed with the goal of providing a tool that can be used reliably and flexibly, but most of all it has focused on avoiding effects that confound foreground modeling efforts in 21cm cosmology.
+
+As such, it has relatively poor performance compared with several other simulators. Deciding which tool to use will ultimately depend on what factors are important to the user. We recommend pyuvsim as a reference tool to verify the results of other simulators, and for producing limited datasets for comparison with observations (particularly on long baselines). Users will need access to a lot of memory and a large number of processor cores to take advantage of pyuvsim's accuracy and precision, ideally on a dedicated high-performance computing cluster.
+
+On this page, we compare pyuvsim's features and performance to several other visibility simulators.
+
+Precision Radio Interferometry Simulator (PRISim)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A python package for generating full-sky simulations.
+
+Advantages:
+
+1. Faster due to vectorized calculations. 
+2. Well-tested through years of use.
+3. Support for a range of sky models including point sources, diffuse emission, and spectral cubes.
+4. Support for tracking as well as transit telescopes.
+5. Parallelized over baselines, frequencies, or sky model/catalog.
+6. Has been tested using imaging with CASA.
+
+Disadvantages:
+
+1. MPI overhead is large if time axis in simulation is long.
+2. Supports only one polarization at a time.
+3. Does not support non-identical antenna patterns.
+
+Fast Holographic Deconvolution (FHD)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+FHD is a data calibration and modeling framework written in IDL by researchers at the University of Washington for analyzing data from the Murchison Widefield Array (MWA) in Western Australia. It calculates sky models by convolving a high-resolution psf with the exact DFT of point sources. This is approach is more efficient when working with wide primary beams, which will be narrower in Fourier space and thus limit the size of the convolution to be performed.
```

Comment:
I think the last sentence is true but it's not a very large effect, so highlighting it so prominently feels a little weird to me.

Reply:
Yeah, and it is not actually accurate compared to other uv-integrators. The sampling pitch of the uv plane _relative_ to the antenna size is what determines the speed, and these are usually chosen at a fixed ratio. So for example a VLA antenna at 40 GHz is very large in wavelengths, but if you pick the uv sampling to go out 2 or 3 sidelobes in the antenna pattern the speed is the same as for the MWA antenna. 

I might also adjust "by researchers at the University of Washington for analyzing data from the Murchison Widefield Array (MWA) in Western Australia." to "by researchers at the University of Washington, originally developed for analyzing EoR data from the Murchison Widefield Array (MWA) in Western Australia."